build/run-me.sh

```diff
@@ -125,6 +138,10 @@ while [ $# -gt 0 ]; do
     continue
   fi
   case $1 in
+  --boot-rot-key)
```

Comment:
I don't like this parameter name.
I think this key is used to sign the FIP image, correct? If so, I think we should use `fip` instead of `boot`.

Reply:
The key is used for more than just signing the FIP image. IIUC, it's used to sign BL2 as well, and it's used to sign various certificates involved in the secure boot chain.